.github/workflows/ci.yml

```diff
@@ -116,6 +116,20 @@ jobs:
         with:
           python-version: '3.8'
           architecture: 'x64'
+      - name: 'Cache Cargo home directory'
```

Comment:
This also includes `tensorboard/data/server/target/` which isn't the Cargo home directory (as I understand it) - maybe broaden description slightly?

Reply:
True, yes. Renamed in both cases. (I think “Cargo artifacts” is probably
fine, since it’s Cargo (not rustc) that sets up this structure. But
happy to entertain other suggestions.)